### PR TITLE
Update index.md

### PR DESCRIPTION
**Pull request by timhowie-okta**
Adding example for `String.startsWith`.

## Description:
- **What's changed?** Added group rules example for String.startsWith.
- **Is this PR related to a Monolith release?**  No

### Resolves:

* [OKTA-567459](https://oktainc.atlassian.net/browse/OKTA-567459)

### CODE REVIEW

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx score **DRAFT 06.01.22**
 
A minimum Acrolinx Score of **40** is required. The total score is an average of the subscores.

Select **Total score** to review the Acrolinx scorecard for your article. Try to increase your individual scores, ex. Correctness. Your article will be clearer and more consistent with Microsoft standards.
 
| Article | **Total score**<br />**Required:70** | Word and phrases<br />(Brand, terms)<br />Preferred: 40 | Correctness<br />(Spelling, grammar)<br />Preferred: 40 | Clarity<br />(Readability)<br />Preferred: 10 | Inclusive language<br />(+ accesibility)<br />Preferred: 10 |
|---------|:------------------------------------:|:--------------------------------------------------------:|:--------------------------------------------------------:|:---------------------------------------------:|:------------------------------------------------------------:|
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/reference/okta-expression-language/index.md](https://github.com/okta/okta-developer-docs/blob/483f33e39f8aa4afba0a43591ac89c3d2b03e7f1/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Freference%2Fokta-expression-language%2Findex.md) | [70](https://okta.acrolinx.cloud/api/v1/checking/scorecards/36994967-4b64-494b-b81b-8a44f90983e5) | 85  | 46  | 72  | 92  |

  - The results from this response were procced by the [hugo-targets1.solution.acrolinx.sh](https://hugo-targets1.solution.acrolinx.sh) Acrolinx Cloud Host.
  - Scorecard links by default are only good for 7 days. [Link for this commit's Content Analysis Dashboard](https://okta.acrolinx.cloud/api/batch/gen.github.0b49f3bc-c25f-41ac-a091-0f21bd4f4465?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

File: packages/@okta/vuepress-site/docs/reference/okta-expression-language/index.md [hunk 1]
```diff
@@ -92,11 +92,12 @@ Okta offers a variety of functions to manipulate attributes or properties to gen
...
 | `String.len`             | (String input)                                                | Integer     | `String.len("This")`                                                                                          | 4              |
 | `String.removeSpaces`    | (String input)                                                | String      | `String.removeSpaces("This is a test")`                                                                       | Thisisatest    |
 | `String.replace`         | (String input, match, replacement)                            | String      | `String.replace("This is a test", "is", "at")`                                                                | That at a test |
 | `String.replaceFirst`    | (String input, match, replacement)                            | String      | `String.replaceFirst("This is a test", "is", "at")`                                                           | That is a test |
+| `String.startsWith`      | (String input, starts)                                        | Boolean     | `String.startsWith("Kiss", "K")`                                                                              | true           |
```
Comment (vanngo-okta) [inline packages/@okta/vuepress-site/docs/reference/okta-expression-language/index.md:100]:
```suggestion
| `String.startsWith`      | (String input, String starts)                                        | Boolean     | `String.startsWith("Kiss", "K")`                                                                              | true           |
```

**Review by vanngo-okta — APPROVED** [review]
Nit suggestion. Approve 🚢

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx score **DRAFT 06.01.22**
 
A minimum Acrolinx Score of **40** is required. The total score is an average of the subscores.

Select **Total score** to review the Acrolinx scorecard for your article. Try to increase your individual scores, ex. Correctness. Your article will be clearer and more consistent with Microsoft standards.
 
| Article | **Total score**<br />**Required:70** | Word and phrases<br />(Brand, terms)<br />Preferred: 40 | Correctness<br />(Spelling, grammar)<br />Preferred: 40 | Clarity<br />(Readability)<br />Preferred: 10 | Inclusive language<br />(+ accesibility)<br />Preferred: 10 |
|---------|:------------------------------------:|:--------------------------------------------------------:|:--------------------------------------------------------:|:---------------------------------------------:|:------------------------------------------------------------:|
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/reference/okta-expression-language/index.md](https://github.com/okta/okta-developer-docs/blob/93ab4a4010f36389a83522caab4840f323a8b419/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Freference%2Fokta-expression-language%2Findex.md) | [70](https://okta.acrolinx.cloud/api/v1/checking/scorecards/8f3de5c2-dd97-443d-a60d-000815729779) | 85  | 46  | 72  | 92  |

  - The results from this response were procced by the [hugo-targets1.solution.acrolinx.sh](https://hugo-targets1.solution.acrolinx.sh) Acrolinx Cloud Host.
  - Scorecard links by default are only good for 7 days. [Link for this commit's Content Analysis Dashboard](https://okta.acrolinx.cloud/api/batch/gen.github.4429443b-2a69-483a-8e60-7be02bdd5a2f?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

**Review by vanngo-okta — APPROVED** [review]
LGTM 🚀

**Pull request merged**